config/initializers/feature_decisions.rb

```diff
@@ -47,5 +47,8 @@
 OpenProject::FeatureDecisions.add :calculated_value_project_attribute,
                                   description: "Allows the use of calculated values as a project attribute."
 
+OpenProject::FeatureDecisions.add :scim_api,
```

Comment:
:exclamation: Please check this again after a rebase. IIRC I added the feature decision _below_ the `block_note_editor`, which means there will not be a git conflict, but our changes effectively do conflict.